dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -123,6 +123,8 @@ def authenticate
       resource_link_id = decoded_jwt[Policies::Lti::LTI_RESOURCE_LINK_CLAIM][:id]
       deployment_id = decoded_jwt[Policies::Lti::LTI_DEPLOYMENT_ID_CLAIM]
       deployment = Queries::Lti.get_deployment(integration.id, deployment_id)
+      nrps_user_type = Policies::Lti.get_account_type(decoded_jwt[Policies::Lti::LTI_ROLES_KEY])
```

Comment:
Since this is in the context of an LTI Launch, NRPS doesn't come into play yet. I would prefer we call this variable `lti_user_type`, or even just `user_type`.

Reply:
Good call! I went with `lti_account_type` to differentiate it against other account types.